commit 8218f7fe9df520a2f5f189632bd869a3f3e38b61
Author: Eric Biggers <ebiggers@google.com>
Date:   Wed Feb 1 21:07:11 2017 -0500

    ext4: fix use-after-iput when fscrypt contexts are inconsistent
    
    commit dd01b690f8f4b1e414f89e5a9a5326bf720d6652 upstream.
    
    In the case where the child's encryption context was inconsistent with
    its parent directory, we were using inode->i_sb and inode->i_ino after
    the inode had already been iput().  Fix this by doing the iput() in the
    correct places.
    
    Note: only ext4 had this bug, not f2fs and ubifs.
    
    Fixes: d9cdc9033181 ("ext4 crypto: enforce context consistency")
    Signed-off-by: Eric Biggers <ebiggers@google.com>
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1280_8218f7fe9df520a2f5f189632bd869a3f3e38b61.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1280_8218f7fe9df520a2f5f189632bd869a3f3e38b61.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
